function-app/code/go.sum

```diff
@@ -57,6 +57,50 @@ github.com/rs/zerolog v1.29.0/go.mod h1:NILgTygv/Uej1ra5XxGf82ZFSLk58MFGAUS2o6us
+github.com/weka/go-cloud-lib v0.0.0-20230620093048-9f3e603fc985/go.mod h1:lT0kVeV0d4HznnITUHbHN6JunpqyL4q9McAYr78YNm4=
+github.com/weka/go-cloud-lib v0.0.0-20230620095340-d3db9bb4f6b1 h1:oE45Km9AxH+ngBK0qFJc48O3kMRHMJq3c+E7qEj/VB8=
+github.com/weka/go-cloud-lib v0.0.0-20230620095340-d3db9bb4f6b1/go.mod h1:lT0kVeV0d4HznnITUHbHN6JunpqyL4q9McAYr78YNm4=
+github.com/weka/go-cloud-lib v0.0.0-20230620095842-ba727ed2be07 h1:ZRFEzhRdVjGYUyOlpagMRpOVbOHA4Uhg+9IH6G7S4ZQ=
+github.com/weka/go-cloud-lib v0.0.0-20230620095842-ba727ed2be07/go.mod h1:lT0kVeV0d4HznnITUHbHN6JunpqyL4q9McAYr78YNm4=
+github.com/weka/go-cloud-lib v0.0.0-20230620121805-cf6e346c4d1e h1:xKBNebN19L1f18ODX66795lRNdz9usbkLNBfd4BZPvo=
+github.com/weka/go-cloud-lib v0.0.0-20230620121805-cf6e346c4d1e/go.mod h1:lT0kVeV0d4HznnITUHbHN6JunpqyL4q9McAYr78YNm4=
+github.com/weka/go-cloud-lib v0.0.0-20230620124543-c2562e75351d h1:QntU4u56DbSztzsiawwA0oGsEC4PHdBpNeqM3RWpCx4=
+github.com/weka/go-cloud-lib v0.0.0-20230620124543-c2562e75351d/go.mod h1:lT0kVeV0d4HznnITUHbHN6JunpqyL4q9McAYr78YNm4=
+github.com/weka/go-cloud-lib v0.0.0-20230620125800-13b091573e6f h1:HuXkqsVYK3yTiFuFdZW8EvzU3oE5ROhX2FiM5Fyg6cI=
+github.com/weka/go-cloud-lib v0.0.0-20230620125800-13b091573e6f/go.mod h1:lT0kVeV0d4HznnITUHbHN6JunpqyL4q9McAYr78YNm4=
+github.com/weka/go-cloud-lib v0.0.0-20230622105851-df3ddfa34271 h1:GHQUT0xMDLiP4q59hJIophOeL4msJhs1uvk0jDSdWq0=
+github.com/weka/go-cloud-lib v0.0.0-20230622105851-df3ddfa34271/go.mod h1:lT0kVeV0d4HznnITUHbHN6JunpqyL4q9McAYr78YNm4=
+github.com/weka/go-cloud-lib v0.0.0-20230622115238-b8114ae63f9b h1:+FiwE9DBMaUOtEHWPKpmM5OvCJg/WJdPQdyGSrq42vw=
+github.com/weka/go-cloud-lib v0.0.0-20230622115238-b8114ae63f9b/go.mod h1:lT0kVeV0d4HznnITUHbHN6JunpqyL4q9McAYr78YNm4=
+github.com/weka/go-cloud-lib v0.0.0-20230622120735-7d74e9181a96 h1:ZwS08eiHt1KFa+9Gu7LFQxL/LSwLsiB5xRTbAMxe2y8=
+github.com/weka/go-cloud-lib v0.0.0-20230622120735-7d74e9181a96/go.mod h1:lT0kVeV0d4HznnITUHbHN6JunpqyL4q9McAYr78YNm4=
+github.com/weka/go-cloud-lib v0.0.0-20230622122219-feb1c51b50d6 h1:4gDlRBV5VWDYiRaHd72fj/mKlooZ2IuuIZZvGegZZH4=
+github.com/weka/go-cloud-lib v0.0.0-20230622122219-feb1c51b50d6/go.mod h1:lT0kVeV0d4HznnITUHbHN6JunpqyL4q9McAYr78YNm4=
+github.com/weka/go-cloud-lib v0.0.0-20230622124207-52128fdad7ff h1:qm9VlAfIFm0eQCI6BdJ2AWcXaHvg/q1g1H7FD7rmb30=
+github.com/weka/go-cloud-lib v0.0.0-20230622124207-52128fdad7ff/go.mod h1:lT0kVeV0d4HznnITUHbHN6JunpqyL4q9McAYr78YNm4=
+github.com/weka/go-cloud-lib v0.0.0-20230622144902-f140365d5662 h1:jhlJM37haptbdxuc8WMpNRhptl132fo27j1d9W+7/+k=
+github.com/weka/go-cloud-lib v0.0.0-20230622144902-f140365d5662/go.mod h1:lT0kVeV0d4HznnITUHbHN6JunpqyL4q9McAYr78YNm4=
+github.com/weka/go-cloud-lib v0.0.0-20230625073119-85d6c80ea543 h1:BwQ4YDjJl/ne+PoGt00wWrGmQZsyacgpzw2Lj/Z4+UQ=
+github.com/weka/go-cloud-lib v0.0.0-20230625073119-85d6c80ea543/go.mod h1:lT0kVeV0d4HznnITUHbHN6JunpqyL4q9McAYr78YNm4=
```

Comment:
you should run `go mod tidy` to clean this

Reply:
good to know